.github/workflows/build.yml

```diff
@@ -73,10 +73,11 @@ jobs:
         
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.2.2
+        if: (!matrix.use_qemu) || fromJSON(env.USE_QEMU)
```

Comment:
It looks like you noticed the change here had unintended effects, more or less what happened is the `if` key applies to the entire step and is position independent within the step -- it's not like other programming languages where it just applies to subsequent lines of code.

```
      - if: (!matrix.use_qemu) || fromJSON(env.USE_QEMU)
        name: Build wheels
        uses: pypa/cibuildwheel@v2.2.2
        env:
          CIBW_ARCHS: "${{ matrix.arch }}"
          CIBW_BUILD: "cp39-${{ matrix.build }}*"
```
and
```
      - name: Build wheels
        uses: pypa/cibuildwheel@v2.2.2
        env:
          CIBW_ARCHS: "${{ matrix.arch }}"
          CIBW_BUILD: "cp39-${{ matrix.build }}*"
        if: (!matrix.use_qemu) || fromJSON(env.USE_QEMU)
```
 would all be equivalent to what you had and skip over the entire step if the condition isn't true.

---

And some info on musllinux/manylinux platform tags:

Building both musllinux and manylinux variants of the wheels for all platforms is fine; those platform tags are unrelated to the CPU (aarch64, s390x, etc) platform tags, and are used to indicate which libc implementation the wheel was built for (musl or glibc).